thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/ThirdEyeAnomalyApplication.java

```diff
@@ -70,6 +73,16 @@ public String getName() {
   @Override
   public void initialize(final Bootstrap<ThirdEyeAnomalyConfiguration> bootstrap) {
     bootstrap.addBundle(new AssetsBundle("/assets/", "/", "index.html"));
+    bootstrap.addBundle(new SwaggerBundle<ThirdEyeConfiguration>() {
```

Comment:
Extract the class of swaggerbundle and reuse it in both anomaly and dashboard applications.